llvm/lib/Transforms/Instrumentation/PGOInstrumentation.cpp

```diff
@@ -1760,17 +1760,10 @@ static void collectComdatMembers(
       ComdatMembers.insert(std::make_pair(C, &GA));
 }
 
-// Don't perform PGO instrumeatnion / profile-use.
-static bool skipPGO(const Function &F) {
+// Return true if we should not annotate this function for PGO
```

Comment:
Nit: how about `Return true if we should not attempt to find an instrumented profile for this function`? (because "annotate" is ambiguous - there are other annotations, I guess you mean profile metadata here? Then, "PGO" really means "instrumented profile use" here)